app/controllers/storytime/subscriptions_controller.rb

```diff
@@ -0,0 +1,39 @@
+require_dependency "storytime/application_controller"
+
+module Storytime
+  class SubscriptionsController < ApplicationController
+    before_action :set_subscription, only: [:destroy]
+
+    def create
+      @subscription = Storytime::Subscription.new(permitted_attributes)
```

Comment:
This isn't setting the site ID and therefore @site.active_email_subscriptions ends up returning nothing. Need to set the site_id on the sub.